x/ibc/07-tendermint/types/test_utils.go

```diff
@@ -22,8 +22,12 @@ func MakeBlockID(hash []byte, partSetSize uint32, partSetHash []byte) tmtypes.Bl
 }
 
 // CreateTestHeader creates a mock header for testing only.
-func CreateTestHeader(chainID string, height, trustedHeight int64, timestamp time.Time, valSet, trustedVals *tmtypes.ValidatorSet, signers []tmtypes.PrivValidator) Header {
-	vsetHash := valSet.Hash()
+func CreateTestHeader(chainID string, height, trustedHeight int64, timestamp time.Time, tmValSet, tmTrustedVals *tmtypes.ValidatorSet, signers []tmtypes.PrivValidator) Header {
```

Comment:
can we move this file and its contents to ibctesting?

Reply:
it's already there but it is a function of a `chain`, it wouldn't make sense to move this specific version there since it should just be deleted once we move to ibctesting in 07-tendermint. I can add it to the wishlist to migrate tendermint tests away from this func so we can delete, but I think that is outside the scope of this pr